src/main/java/com/ldtteam/structurize/placement/structure/CreativeStructureHandler.java

```diff
@@ -140,6 +140,12 @@ public void prePlacementLogic(final BlockPos worldPos, final BlockState blockSta
     @Override
     public BlockState getSolidBlockForPos(final BlockPos worldPos)
     {
-        return BlockUtils.getSubstitutionBlockAtWorld(getWorld(), worldPos);
+        return BlockUtils.getSubstitutionBlockAtWorld(getWorld(), worldPos, null);
+    }
+
+    @Override
+    public BlockState getWorldGenBlock(final BlockPos worldPos, final BlockState virtualBlockAbove)
+    {
+        return BlockUtils.getSubstitutionBlockAtWorld(getWorld(), worldPos, virtualBlockAbove);
     }
```

Comment:
I don't get why that has to be an extra method. That should just be called by the default getSolidBlockForPos behaviour, but not have to exist in all handlers.

Reply:
name addressed, leaving the method where it is - not gonna touch tape